package.json

```diff
@@ -67,6 +67,7 @@
     "@stripe/react-stripe-js": "^1.1.2",
     "css-loader": "^5.0.0",
     "debug": "^4.1.1",
+    "interpolate-components": "^1.1.1",
```

Comment:
It was a surprise to me that we didn't declare this dependency until now and I thought this is the first usage of `interpolate-components` within the project 😮 

Now I see that it was already required through `@woocommerce/components`, but explicitly declaring it is the better option 👍

Reply:
There is a usage of `__experimentalCreateInterpolateElement` [here](https://github.com/Automattic/woocommerce-payments/blob/add%2F897-tos-modal/client/account-status/index.js#L8), but it's not the same IMO. I think it makes sense to create an issue to replace `__experimental...` with `interpolate-components`. What do you think?

Reply:
I agree. This was not a change request ;)